src/App.tsx

```diff
@@ -1,9 +1,22 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [searchMovie, setSearchMovie] = useState('');
+
+  const isInclude = (value: string) => Boolean(
+    value.toLowerCase().includes(
+      searchMovie.toLowerCase().trim(),
```

Comment:
also you can delete unnecessary spaces between words in search query. If I search `love             actually` I will not get results.

Reply:
fixed